lib/logger/utils.ts

```diff
@@ -322,11 +322,15 @@ export function validateLogLevel(
 }
 
 // Can't use `util/regex` because of circular reference to logger
-const urlRe = /[a-z]{3,9}:\/\/[^@/]+@[a-z0-9.-]+/gi;
+const urlRe = /(?:[a-z]{3,9}:\/\/[^@/]+@[a-z0-9.-]+|data:[^,]+,[^"]*)/gi;
```

Comment:
Revert this change, instead add extra "if" logic later to specifically handle dataUri's

Reply:
Done